meta/recipes-devtools/python/python3-brotli_1.0.9.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Brotli compression format"
+SECTION = "devel/python"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/brotli-${PV}/LICENSE;md5=941ee9cd1609382f946352712a319b4b"
```

Comment:
`${WORKDIR}/brotli-${PV}` is not really needed here, as `${S}` is the base path for searching these files